webui/react/package.json

```diff
@@ -122,15 +122,16 @@
     "magic-string": "^0.27.0",
     "morgan": "^1.10.0",
     "npm-force-resolutions": "0.0.10",
+    "postcss": "^8.4.21",
```

Comment:
Where do we use this?

Reply:
stylelint v14 required `postcss` with `TypeError: Class extends value undefined is not a constructor or null` error. However, it seems like v15 doesnt require it. uninstalled it. nice finding